### DISCUSSION TITLE
Help with a Handhels

### DISCUSSION THREAD

**Discussion by CarlosWss** [OnionUI/Onion#1556] [General]
I find (literaly) a handheld called X6 (is like a PSP), and I open it, it use a ATj2273c Soc from Actions Semiconductor, will the Onion OS run on it?
![image](https://github.com/OnionUI/Onion/assets/53383610/c156a309-e6ef-4d76-9ac1-3329ddb4967f)

**Comment by Aemiii91** [comment]
No
